databroker/src/broker.rs

```diff
@@ -179,6 +180,23 @@ pub struct DataBroker {
     shutdown_trigger: broadcast::Sender<()>,
 }
 
+#[async_trait::async_trait]
+pub trait ActuationProvider {
+    async fn actuate(&self, actuation_changes: Vec<ActuationChange>) -> Result<(), tonic::Status>;
+}
+
+#[derive(Clone)]
+pub struct ActuationChange {
+    pub id: i32,
+    pub data_value: DataValue,
+}
+
+pub struct ActuationSubscription {
+    vss_ids: Vec<i32>,
+    actuation_provider: Box<dyn ActuationProvider + Send + Sync + 'static>,
+    permissions: Permissions,
+}
```

Comment:
right now some cleanup operation is missing, e.g. an ActuationProvider disconnects or crashes and can no longer actuate. However, we prevent multiple ActuationProviders to register for the same signal, meaning once the ActuationProvider disconnects, it can no longer re-register to it's signal. 

I can not use the `actuation_provider` to check if it is still available. I maybe could check (e.g. Err during send()) in val.rs -> Provider -> actuate but there I would not have the relation between the ActuationProvider and the ActuationSubscription. Sounds like a lot of overhead and not sure if there isn't a better way